cloudpebble/settings.py

```diff
@@ -144,7 +145,7 @@
 )
 
 BOWER_INSTALLED_APPS = (
-    'https://github.com/krisk/Fuse.git#2c1560d763',
+    'https://github.com/krisk/Fuse.git',
```

Comment:
I changed it because I wanted fixes in the latest commit - I'll go ahead and fix it to the current one